src/vea/tests/vea_ut.c

```diff
@@ -349,7 +345,8 @@ ut_reserve_bitmap(void **state)
 
 	/* correctness check */
 	ext = d_list_entry(r_list->prev, struct vea_resrvd_ext, vre_link);
-	assert_int_equal(ext->vre_hint_off, VEA_HINT_OFF_INVAL);
+	/* 1024 + 261 */
+	assert_int_equal(ext->vre_hint_off, 0x505);
```

Comment:
I don't see where this "1024 + 261" come from. Is it depending on previous sub-tests?